pkg/build/controller/build/build_controller.go

```diff
@@ -236,6 +242,15 @@ func (bc *BuildController) Run(workers int, stopCh <-chan struct{}) {
 	defer bc.buildQueue.ShutDown()
 	defer bc.buildConfigQueue.ShutDown()
 
+	// Read additionalCA data, if it exists
+	if len(bc.additionalTrustedCAPath) > 0 {
+		caData, err := bc.readBuildCAData()
+		if err != nil {
+			glog.Warningf("Failed to read additional CA bundle %s: %v", bc.additionalTrustedCAPath, err)
+		}
```

Comment:
For now build controller doesn't fail if the additional CA can't be read.
This could cause pain later on if image pulls/pushes fail (and would be a challenge to diagnose). Would we rather have all builds fail in this scenario?

Reply:
I don't know that failing all the builds will make it easier to diagnose... the builds will fail to pull/push because they don't trust the registry and it's a pretty short path from there to "ok, what CAs are being setup for the builds", so i'd prefer to just log it rather than use failing builds as a way to communicate cluster state.

It would also expose internal infrastructure issues (that might not even be relevant to all builds) to all users of the system.

In any case this should be an exceptionally unusual condition, unless the admin has manually set the path incorrectly (and if they're setting the path, they're already in an unsupported situation).